apps/reaper/cameras.xml

```diff
@@ -0,0 +1 @@
ageUpdateInterval></CameraFeed></CameraFeeds></CameraSite><CameraSite><Location><![CDATA[US-35 at US-62]]></Location><Latitude>39.523798</Latitude><Longitude>-83.460089</Longitude><CameraFeeds><CameraFeed><Direction>PTZ</Direction><Type>Pan/Tilt/Zoom</Type><Description><![CDATA[US-35 at US-62.  This is a Pan/Tilt/Zoom camera.  The camera view direction is variable and may not be relative to its current position.  This image updates every 30 seconds.   If this camera image is not currently updating or displaying properly, ODOT personnel are aware of the situation and are working to address the issue.]]></Description><SmallImage><![CDATA[http://cmhimg01.dot.state.oh.us/images/CMH/RWIS_D06-2.jpg]]></SmallImage><LargeImage><![CDATA[http://cmhimg01.dot.state.oh.us/images/CMH/RWIS_D06-2.jpg]]></LargeImage><ImageUpdateInterval>10</ImageUpdateInterval></CameraFeed></CameraFeeds></CameraSite><CameraSite><Location><![CDATA[I-75 at Dowling Rd]]></Location><Latitude>41.477410</Latitude><Longitude>-83.621830</Longitude><CameraFeeds><CameraFeed><Direction>PTZ</Direction><Type>Pan/Tilt/Zoom</Type><Description><![CDATA[I-75 at Dowling Rd.  This is a Pan/Tilt/Zoom camera.  The camera view direction is variable and is indicated in the top left corner of this image (N, S, E, W, NE, NW, SE, SW).   If this camera image is not currently updating or displaying properly, ODOT personnel are aware of the situation and are working to address the issue.]]></Description><SmallImage><![CDATA[http://cmhimg01.dot.state.oh.us/images/toledo/I-75_at_Dowling_Rd.jpg]]></SmallImage><LargeImage><![CDATA[http://cmhimg01.dot.state.oh.us/images/toledo/I-75_at_Dowling_Rd.jpg]]></LargeImage><ImageUpdateInterval>5</ImageUpdateInterval></CameraFeed></CameraFeeds></CameraSite><CameraSite><Location><![CDATA[I-77 at 4th St]]></Location><Latitude>40.801972</Latitude><Longitude>-81.398287</Longitude><CameraFeeds><CameraFeed><Direction>PTZ</Direction><Type>Pan/Tilt/Zoom</Type><Description><![CDATA[I-77 at 4th St.  This is a Pan/Tilt/Zoom camera.  The camera view direction is variable and may not be relative to its current position. If this camera image is not currently updating or displaying properly, ODOT personnel are aware of the situation and are working to address the issue.]]></Description><SmallImage><![CDATA[http://cmhimg01.dot.state.oh.us/images/CLE/CLE120a.jpg]]></SmallImage><LargeImage><![CDATA[http://cmhimg01.dot.state.oh.us/images/CLE/CLE120a.jpg]]></LargeImage><ImageUpdateInterval>5</ImageUpdateInterval></CameraFeed></CameraFeeds></CameraSite><CameraSite><Location><![CDATA[I-75 at Stanley Ave]]></Location><Latitude>39.788843</Latitude><Longitude>-84.183595</Longitude><CameraFeeds><CameraFeed><Direction>PTZ</Direction><Type>Pan/Tilt/Zoom</Type><Description><![CDATA[I-75 at Stanley Ave.  This is a Pan/Tilt/Zoom camera.  The camera view direction is variable and is indicated in the top left corner of this image (N, S, E, W, NE, NW, SE, SW).   If this camera image is not currently updati
```

Comment:
Should these reaper files be part of this andi card?

Reply:
Got them in the rebase